client/deliveries.lua

```diff
@@ -41,7 +41,7 @@ local function OpenDealerShop()
     repItems.items = {}
     repItems.slots = 30
     for k, _ in pairs(Config.Dealers[currentDealer].products) do
-        if QBCore.Functions.GetPlayerData().metadata.dealerrep >= Config.Dealers[currentDealer].products[k].minrep then
+        if exports.qbx_core:GetPlayerData().metadata.dealerrep >= Config.Dealers[currentDealer].products[k].minrep then
```

Comment:
use QBX.PlayerData rather than call an export for this. PlayerData should come from the playerdata module only

Reply:
oh my bad i didnt noticed i'll update it